zeebe/engine/src/test/java/io/camunda/zeebe/engine/state/message/PendingProcessMessageSubscriptionStateTest.java

```diff
@@ -27,27 +29,33 @@ public final class PendingProcessMessageSubscriptionStateTest {
   @Rule public final ProcessingStateRule stateRule = new ProcessingStateRule();
 
   private MutableProcessMessageSubscriptionState persistentState;
-  private PendingProcessMessageSubscriptionState transientState;
+  private PendingProcessMessageSubscriptionState pendingSubscriptionState;
+  private TransientPendingSubscriptionState transientSubscriptionState;
 
   @Before
   public void setUp() {
     persistentState = stateRule.getProcessingState().getProcessMessageSubscriptionState();
-    transientState = stateRule.getProcessingState().getPendingProcessMessageSubscriptionState();
+    pendingSubscriptionState =
+        stateRule.getProcessingState().getPendingProcessMessageSubscriptionState();
+    transientSubscriptionState =
+        stateRule.getProcessingState().getTransientPendingSubscriptionState();
   }
 
   @Test
   public void shouldNoVisitSubscriptionBeforeTime() {
     // given
     final ProcessMessageSubscriptionRecord record1 = subscriptionRecordWithElementInstanceKey(1L);
     persistentState.put(1L, record1);
-    transientState.onSent(record1, 1_000L);
+    addToTransientSubscriptionState(record1);
+    pendingSubscriptionState.onSent(record1, 1_000L);
```

Comment:
🔧 We might consider combining the call to `persistentState.put` and `addToTransientSubscriptionState` into a single method, and similarly group together the other operations to clarify which operations we expect to happen together. But I'm alright if we forgo that now.